utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -0,0 +1,209 @@
+	EOF
+}
+
+function run
+{
+	if [[ $TRACE_LEVEL -ge $TRACE_LEVEL_STANDARD ]]
+	then
+		"$@"
+	else
+		"$@" &> /dev/null
+	fi
+}
+
+OPTIONS=$(getopt -o "j:fhVvDq" --long "jobs:,force,help,version,verbose,debug,quiet" -- "$@") || exit 1
+eval set -- "$OPTIONS"
+while true
+do
+	case "$1" in
+		-j|--jobs) JOBS_NB="$2" ; shift 2 ;;
+		-f|--force) FORCE_INSTALL=true ; shift 1;;
+		-h|--help) usage ; exit 0;;
+		-V|--version) echo "daos-make.sh version=$VERSION" ; exit 0 ;;
+		-v|--verbose) TRACE_LEVEL=$TRACE_LEVEL_VERBOSE ; shift 1 ;;
+		-D|--debug) TRACE_LEVEL=$TRACE_LEVEL_DEBUG ; set -x ; shift 1 ;;
+		-q|--quiet) TRACE_LEVEL=$TRACE_LEVEL_QUIET ; shift 1 ;;
+		--) shift ; break ;;
+		*) fatal "unrecognized command line option" ;;
+	esac
+done
+
+[[ $1 ]] || fatal "Command not defined: install, update"
+CMD="$1"
+
+check_cmds $SCONS_EXE
+
+set -e
+
+info "Mount of home partition"
+{% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
+run $CLUSH_EXE -l root -w $CLIENTS_LIST -w $SERVERS_LIST  mountpoint "$HOME"
+{% else %}
+run $CLUSH_EXE -w $SERVERS_LIST ldcondig mountpoint "$HOME"
+{% endif %}
+
+for dir in "$DAOS_INSTALL_DIR" "$DAOS_SOURCE_DIR/build"
+do
+	if [[ -d "$dir" && "$CMD" == install ]]; then
+		[[ -n "$FORCE_INSTALL" ]] || fatal "DAOS already installed: use install --force or update."
+		info "Removing old DAOS install directory $dir"
+		if ! run /bin/rm -fr "$dir" ; then
+			fatal "Old install directory $dir could not be removed"
+		fi
+
+	fi
+done
+
+if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
+	info "Creating DAOS install directory $DAOS_INSTALL_DIR"
+	if ! run mkdir -p "$DAOS_INSTALL_DIR" ; then
+		fatal "DAOS install directory $DAOS_INSTALL_DIR could not be created"
+	fi
+fi
+
+cd "$DAOS_SOURCE_DIR"
+SCONS_OPTS="-j $JOBS_NB"
+case $CMD in
+	install) SCONS_OPTS="$SCONS_OPTS --build-deps=yes --config=force" ;;
+	update) SCONS_OPTS="$SCONS_OPTS --build-deps=no" ;;
+	*) fatal "Unexpected case" ;;
+esac
+info "Building and installing DAOS from source tree $PWD to $DAOS_INSTALL_DIR"
+if ! run env MPI_PKG=any "$SCONS_EXE" BUILD_TYPE=debug PREFIX="$DAOS_INSTALL_DIR" install $SCONS_OPTS ; then
+	fatal "DAOS could not be properly build and/or installed"
+fi
+
+info "Finalizing spdk install"
+for item in $(find "$DAOS_INSTALL_DIR/share/daos" -type f)
+do
+        dest="/usr/${item#$DAOS_INSTALL_DIR/}"
+        run $CLUSH_EXE -l root -w $SERVERS_LIST install -v -D -o root -g root -m 755 "$item" "$dest"
+done
+
+info "Finalizing daos-admin install"
+run chmod 0446 "$DAOS_INSTALL_DIR/bin/daos_admin"
+run $CLUSH_EXE -l root -w $SERVERS_LIST cp -f "$DAOS_INSTALL_DIR/bin/daos_admin" /usr/bin
+run $CLUSH_EXE -l root -w $SERVERS_LIST chown root:daos_server /usr/bin/daos_admin
+run $CLUSH_EXE -l root -w $SERVERS_LIST chmod 4755 /usr/bin/daos_admin
+run $CLUSH_EXE -w $SERVERS_LIST chmod a-x  "$DAOS_INSTALL_DIR/bin/daos_admin"
+
+{% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
```

Comment:
(lint) The mentioned syntax error was in this brace group. [SC1009]
(lint) You need a space after the '{'. [SC1054]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]